src/plugins/intel_cpu/src/node.cpp

```diff
@@ -1028,14 +1028,19 @@ const std::vector<impl_desc_type>& Node::getDefaultImplPriority() {
         impl_desc_type::jit_sse42_dw,
         impl_desc_type::jit_sse42_1x1,
         impl_desc_type::jit_sse42,
+#if defined(OPENVINO_ARCH_ARM64)
+        impl_desc_type::jit_asimd,
+#endif
         impl_desc_type::gemm_any,
         impl_desc_type::gemm_blas,
         impl_desc_type::gemm_avx512,
         impl_desc_type::gemm_avx2,
         impl_desc_type::gemm_avx,
         impl_desc_type::gemm_sse42,
+#if defined(OV_CPU_WITH_ACL)
```

Comment:
What happens if we do not put it under ifdef here?

Reply:
we don't need ACL support unless `OV_CPU_WITH_ACL` is defined

Reply:
I mean the similar functions in convoluiton / fullyconnected nodes do not have similiar entries under compile time flag.
I think we should have them aligned.

Reply:
aligned (rollbacked)